aclk/README.md

```diff
@@ -84,7 +84,9 @@ Features:
 
 To verify which ACLK implementation Netdata uses, visit the `/api/v1/info` endpoint on your local dashboard and check the `aclk-implementation` key.
 
-New Netdata Cloud features will be implemented on top of ACLK-NG from this point on. ACLK Legacy is therefore kept as a fallback in case some users have issues with ACLK-NG or need to use features which are not yet available in ACLK-NG *(like IPv6 support and SOCKS proxy)*.
+New Netdata Cloud features will be implemented on top of ACLK-NG only. ACLK Legacy is therefore kept as a fallback in case some users have issues.
+
+ACLK Legacy is planned to be removed in following releases!
```

Comment:
```suggestion

> Note: ACLK Legacy will be removed in following releases! 
> Update your ACLK configuration to ACLK-NG to prevent any disruptions.

```
Put this sentence into a Note for higher visibility.